cmd/monaco/main.go

```diff
@@ -17,22 +17,14 @@
 package main
 
 import (
-	"encoding/json"
-	"flag"
 	"fmt"
-	"math/rand"
+	"log"
 	"os"
-	"strconv"
-	"strings"
-
-	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/api"
-	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/config"
-	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/delete"
-	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/environment"
-	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/project"
-	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/rest"
+
 	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/util"
 	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/version"
+
+	"github.com/urfave/cli/v2"
```

Comment:
I really like this CLI. Compared to cobra it produces more readable code, awesome! :rocket:

Reply:
Yeah I had a look at cobra and to be honest it seems way
to complicated for what it is trying to achieve.